admin/class-system-dashboard-admin.php

```diff
@@ -1246,7 +1246,9 @@ public function sd_server_location() {
 
 		}
 
-		return $location;
+		$location = trim(trim($location),',');
```

Comment:
Thank you. I wonder if both city and country are detected, would this remove the comma between the two? At present I'm modifying the code based on your hint so that comma is preserved in that scenario, but the function will also return 'Undetectable' when both city and country are not detected.

Reply:
it is only removed if the city or country is not detected.

**Before patch:**
![image](https://user-images.githubusercontent.com/1203635/161433118-b14d69dd-9b95-475a-bc56-81d8fa1553e5.png)

**After patch:**
![image](https://user-images.githubusercontent.com/1203635/161433145-d4c3e7e4-a143-43f0-8eca-67e218dafccf.png)